src/main/groovy/com/modelsolv/reprezen/soapui/RepreZenExporter.groovy

```diff
@@ -133,10 +139,13 @@ class RepreZenExporter {
 		}
 
 		resource.childResourceList.each {
-			ResourceDefinition res = createResourceDefinition( it, resourceAPI )
+			ResourceDefinition res = createResourceDefinition( it, resourceAPI, dataModel)
```

Comment:
I'm inferring from this that WADL sub-resources are called "child resources" in this library.  So we're talking about a recursive hierarchy of resources, which RAPID-ML doesn't support.

I'm looking for some evidence that we're handling child resource URIs appropriately, and I can't find it.  I looked at the createResourceDefinition method, and I don't see logic to prepend the URI of the parent resource.  Am I missing something, or is this an import bug?